frontend/sw360-portlet/src/main/webapp/html/components/detailRelease.jsp

```diff
@@ -58,26 +58,52 @@
-        <sw360:DisplaySubscribeButton email="<%=themeDisplay.getUser().getEmailAddress()%>" object="${release}" id="SubscribeButton" />
-    </span>
-    </label>
-    </p>
+      <div id="header">
+        <p class="pageHeader"> <span class="pageHeaderBigSpan"> Component: ${component.name}</span>
+
+            <span class="dropdown">
+                <core_rt:forEach var="releaseItr" items="${component.releases}">
+                    <core_rt:if test="${releaseItr.id == releaseId}"> <button onclick="clickOnDropdown()" class="dropbtn"> ${releaseItr.name} ${releaseItr.version} <span class="arrow-down"/>  </button> </core_rt:if>
+                </core_rt:forEach>
+                <span id="releaseDropdown" class="dropdown-content">
+                <core_rt:forEach var="releaseItr" items="${component.releases}">
+                    <a href='<portlet:renderURL ><portlet:param name="<%=PortalConstants.COMPONENT_ID%>" value="${component.id}"/><portlet:param name="<%=PortalConstants.RELEASE_ID%>" value="${releaseItr.id}"/><portlet:param name="<%=PortalConstants.PAGENAME%>" value="<%=PortalConstants.PAGENAME_RELEASE_DETAIL%>"/></portlet:renderURL>' target='_self'>${releaseItr.name} ${releaseItr.version}
+                        <core_rt:if test="${releaseItr.clearingState.value == 4}">  <span class="sw360CircleOK"> </span> </core_rt:if>
+                        <core_rt:if test="${releaseItr.clearingState.value == 0}"> <span class="sw360CircleAlert"> </span> </core_rt:if>
+                        <core_rt:if test="${releaseItr.clearingState.value == 1}">  <span class="sw360CircleAlert"> </span> </core_rt:if>
+                        <core_rt:if test="${releaseItr.clearingState.value == 2}">  <span class="sw360CircleWarning"> </span> </core_rt:if>
+                        <core_rt:if test="${releaseItr.clearingState.value == 3}">  <span class="sw360CircleWarning"> </span> </core_rt:if>
+                    </a>
+                </core_rt:forEach>
+                </span>
+            </span>
+            <span class="pull-right">
+                <input type="button" id="edit" data-release-id="${releaseId}" value="Edit" class="addButton">
+                <sw360:DisplaySubscribeButton email="<%=themeDisplay.getUser().getEmailAddress()%>" object="${release}" id="SubscribeButton" />
+            </span>
+        </p>
+        </div>
+
     <core_rt:set var="inReleaseDetailsContext" value="true" scope="request"/>
     <%@include file="/html/components/includes/releases/detailOverview.jspf"%>
 </core_rt:if>
 <script>
+    function clickOnDropdown() {
+       document.getElementById("releaseDropdown").classList.toggle("show");
+    }
+
+    // Close the dropdown if the user clicks outside of it
+    window.onclick = function(event) {
+        if (!event.target.matches('.dropbtn')) {
+
+            var dropdown = document.getElementsByClassName("dropdown-content");
+            var i;
+
+            dropdown.classList.remove('show');
```

Comment:
this will result in a type error undefined
`dropdown.classList is undefined`
your dropdown variable is still an array.